exercises/rail-fence-cipher/tests/rail-fence-cipher.rs

```diff
@@ -0,0 +1,97 @@
+//! Tests for rail-fence-cipher
+//!
+//! Generated by [script][script] using [canonical data][canonical-data]
+//!
+//! [script]: https://github.com/exercism/rust/blob/master/bin/init_exercise.py
+//! [canonical-data]: https://raw.githubusercontent.com/exercism/problem-specifications/master/exercises/rail-fence-cipher/canonical_data.json
+//!
+//! The tests do not expect any normalization or cleaning.
+//! That trade is tested in enough other exercises.
+
+extern crate rail_fence_cipher;
+use rail_fence_cipher::*;
+
+/// Process a single test case for the property `encode`
+///
+/// All cases for the `encode` property are implemented
+/// in terms of this function.
+fn process_encode_case(input: &str, rails: u32, expected: &str) {
+    let rail_fence = RailFence::new(rails);
+    assert_eq!(rail_fence.encode(input), expected,);
```

Comment:
the trailing comma seems unusual for situations where the closing parenthesis is on the same line